doc/reference/storage_drivers.md

```diff
@@ -67,9 +69,15 @@ Btrfs, ZFS and Ceph RBD have an internal send/receive mechanism that allows for
 LXD uses this optimized transfer when transferring instances and snapshots between storage pools that use the same storage driver, if the storage driver supports optimized transfer and the optimized transfer is actually quicker.
 Otherwise, LXD uses `rsync` to transfer container and file system volumes, or raw block transfer to transfer virtual machine and custom block volumes.
 
-The optimized transfer uses the underlying storage driver's native functionality for transferring data, which is usually faster than using `rsync`.
-However, the full potential of the optimized transfer becomes apparent when refreshing a copy of an instance or custom volume that uses periodic snapshots.
-With optimized transfer, LXD bases the refresh on the latest snapshot, which means:
+The optimized transfer uses the underlying storage driver's native functionality for transferring data, which is usually faster than using `rsync` or raw block transfer.
+
+(storage-optimized-volume-refresh)=
+### Optimized volume refresh
+
+The full potential of the optimized transfer becomes apparent when refreshing a copy of an instance or custom volume that uses periodic snapshots.
+If the optimized transfer isn't supported by the driver or refresh, instead of the delta, the entire volume including its snapshot(s) will be copied using either `rsync` or raw block transfer. LXD will try to keep the overhead low by transferring only the volume itself or any snapshots that are missing on the target.
+
+When refreshing an instance or custom volume, LXD bases the refresh on the latest snapshot, which means:
```

Comment:
I think we need to make it clearer here that we're now talking about the case where optimized refresh is supported.

For example:
```suggestion
When optimized refresh is available for an instance or custom volume, LXD bases the refresh on the latest snapshot, which means:
```

Reply:
Thanks, much clearer, fixed.